sdk/communication/communication-chat/README.md

```diff
@@ -57,7 +57,7 @@ Use the `createThread` method to create a chat thread.
 - Use `topic` to give a thread topic;
 - Use `participants` to list the chat participants to be added to the thread;
 
-`createChatThreadResponse` is the response returned from creating a thread. It contains a `chatThread` which is the thread that was created, as well as an `errors` property which will contain information about invalid participants if they failed to be added to the thread.
+`CreateChatThreadResult` is the response returned from creating a thread. It contains a `chatThread` which is the thread that was created, as well as an `errors` property which will contain information about invalid participants if they failed to be added to the thread.
```

Comment:
createChatThreadResult(lower case the first one - that is the name on the example)